.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,30 @@
+name: CI Automate
+on:
+  push:
+    branches:
+      - '**'
```

Comment:
If we are targeting all branches, you just provide no argument. It will run on all branches.

Reply:
Sure